.xcode/EnumPropertyWithAssociatedData.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+  <key>IDECodeSnippetCompletionPrefix</key>
+  <string>enumproperty</string>
+  <key>IDECodeSnippetCompletionScopes</key>
+  <array>
+    <string>All</string>
+  </array>
+  <key>IDECodeSnippetContents</key>
+  <string>  public var &lt;#propertyName#&gt;: &lt;#AssociatedType#&gt;? {
```

Comment:
made this public because even if the enum is internal or private Xcode doesn't generate a warning, so it seems this gives the most unsurprising behavior of enum properties.